init.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+'use strict';
+const execa = require('execa');
+const npmRunPath = require('npm-run-path');
+
+const env = npmRunPath.env({cwd: __dirname});
+
+execa('alfred-link', {env}).catch(err => {
```

Comment:
Actually, I just realized something. `execa` includes `npm-run-path` and work on process.cwd(), but if you pass in a custom `cwd` to `execa`, it will also be passed to `npm-run-path`, so I'm pretty sure all you have to do is to pass `{cwd: __dirname}` to `execa` and it should work.